.env.development

```diff
@@ -2,6 +2,7 @@
 # Be careful not the check in any secret API keys
 
 REACT_APP_API_ENDPOINT=https://staging-scl.herokuapp.com/
+# REACT_APP_API_ENDPOINT=http://localhost:5000/ # swap this for local testing
```

Comment:
dont forget to revert before merging :)

Reply:
I decided to leave this in as a comment in case someone wanted it! I can remove it though if it feels unnecessary :)